docs/docs/reference/slsa.md

```diff
@@ -0,0 +1,73 @@
+# Supply chain levels for software artifacts (SLSA) adoption
+
+[Supply chain Levels for Software Artifacts, or SLSA (salsa)](https://slsa.dev/) is a framework that provides incremental improvements to a projects build system and engineering processes. SLSA is structured in [four levels](https://slsa.dev/spec/v0.1/levels). This page describes the adoption of SLSA for Constellation.
+
+:::info
+SLSA is still in alpha. The presented levels and their requirements might change in future. We will adopt any changes into our engineering processes, as they get defined.
+:::
+
+## Level 1 - Adopted
+
+**[Build - Scripted](https://slsa.dev/spec/v0.1/requirements#scripted-build)**
+
+All build steps are automated via [CMake](https://github.com/edgelesssys/constellation/blob/main/CMakeLists.txt) and [GitHub Actions](https://github.com/edgelesssys/constellation/tree/main/.github).
+
+**[Provenance - Available](https://slsa.dev/spec/v0.1/requirements#available)**
+
+Provenance for our CLI is generated using the [slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator).
+
+## Level 2 - Adopted
+
+**[Source - Version Controlled](https://slsa.dev/spec/v0.1/requirements#version-controlled)**
+
+Constellation is hosted on GitHub using git.
+
+**[Build - Build Service](https://slsa.dev/spec/v0.1/requirements#build-service)**
+
+All builds are carried out by [GitHub Actions](https://github.com/edgelesssys/constellation/tree/main/.github).
+
+**[Provenance - Authenticated](https://slsa.dev/spec/v0.1/requirements#authenticated)**
+
+Provenance for our CLI is signed using the [slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator). Learn [how to verify the CLI](../workflows/verify-cli.md) using the signed provenance, before using it for the first time.
+
+**[Provenance - Service Generated](https://slsa.dev/spec/v0.1/requirements#service-generated)**
+
+Provenance for our CLI is generated using the [slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator) in GitHub Actions.
+
+## Level 3 - Adopted
+
+**[Source - Verified History](https://slsa.dev/spec/v0.1/requirements#verified-history)**
+
+The [Edgeless Systems](https://github.com/edgelesssys) GitHub organization [requires two-factor authentication](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-two-factor-authentication-for-your-organization/requiring-two-factor-authentication-in-your-organization) for all members.
+
+**[Source - Retained Indefinitely](https://slsa.dev/spec/v0.1/requirements#retained-indefinitely)**
+
+Since we use GitHub to host our repository, it's not possible for an external person to delete or modify the history. Before a pull request can be merged, an explicit approval from an [Edgeless Systems](https://github.com/edgelesssys) team member is required.
```

Comment:
```suggestion
Since we use GitHub to host the repository, it's impossible for an external person to delete or modify the history. Before a pull request can be merged, an explicit approval from an [Edgeless Systems](https://github.com/edgelesssys) team member is required.
```